### PR TITLE
1158 language.isSignLanguage

### LINKED ISSUE #1158: Language.isSignLanguage boolean field
Add `isSignLanguage` boolean field to Language. It can assigned at create & update. defaults to false.

### PR DESCRIPTION
**Pull request by tjgersho**
Closes #1158

### CODE REVIEW

File: src/components/language/dto/update-language.dto.ts [hunk 1]
```diff
@@ -72,6 +72,9 @@ export abstract class UpdateLanguage {
 
   @NameField({ nullable: true })
   readonly leastOfTheseReason?: string;
+
+  @Field({ nullable: true })
+  readonly isSignLanguage?: boolean = false;
```
Comment (CarsonF) [inline src/components/language/dto/update-language.dto.ts:77]:
```suggestion
  readonly isSignLanguage?: boolean;
```
We don't want it set back to false if it's omitted. Probably copy/paste error 😉

**Review by CarsonF — CHANGES_REQUESTED** [review]
Just that one thing and the conflicts, then good to go

File: src/components/language/dto/update-language.dto.ts (hunk 1 above)
Comment (tjgersho) [reply src/components/language/dto/update-language.dto.ts:77]: 🤦

**Review by CarsonF — APPROVED** [review]
Thanks @tjgersho!

**Pull request merged**